packages/forma-36-react-components/src/components/Icon/svg/DoubleArrow.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+function SvgDoubleArrow(props: React.SVGProps<SVGSVGElement>) {
+  return (
+    <svg width="1em" height="1em" viewBox="0 0 24 24" {...props}>
+      <path d="M 7 14 L 12 20 L 17 14 Z" />
+      <path d="M 6.989 9.951 L 11.989 4 L 16.989 9.951 Z" />
```

Comment:
@mshaaban0 the second path doesn't look "optimized" 🤷‍♂️ . If there's a way to make it better go nuts!

Reply:
Looks like the third `<path/>` which is just there for spacing, and decimals on second path are caused by how the icon was designed/exported from the design tool, can be improved but not a big deal :)